docs/configuring-a-resource.md

```diff
@@ -0,0 +1,410 @@
+## Configuring a Resource
+
+[Terrajet] generates as much as it could using the available information in the
+Terraform resource schema. This includes an XRM-conformant schema of the
+resource, controller logic, late initialization, sensitive data handling etc.
+However, there are still couple of information that requires some input
+configuration which could easily be provided by checking the Terraform
+documentation of the resource:
+
+- [External name]
+- [Cross Resource Referencing]
+- [Additional Sensitive Fields and Custom Connection Details]
+- [Late Initialization Behavior]
+
+### External Name
```

Comment:
I think this section needs an update because of https://github.com/crossplane-contrib/terrajet/pull/132

Reply:
Opened a ticket for that: https://github.com/crossplane-contrib/terrajet/issues/153